doc/source/parameters/resolved_cfd-dem/resolved_cfd-dem.rst

```diff
@@ -81,26 +87,26 @@ This subsection contains the parameters related to the resolved CFD-DEM around p
 
-To sharpen the immersed boundary of each particles, a layer of cells around the immersed boundary can be refined forming a hypershell of refined cells.
+To sharpen the immersed boundary of each particles, a layer of cells around the immersed boundary can be refined forming a near-particle zone of refined cells. An effective radius, dependent on the shape, is calculated at the shape initialization. Its definition is chosen so that the refinement zone is close to the surface. For example, a torus' effective radius is the ring thickness radius. For a rectangle (or box), it's the mean of the half-lengths. For a sphere, it's the actual radius.
 
-* The ``refine mesh inside radius factor`` parameter defines the inside radius of the hypershell that forms the refinement zone around the particles. The radius used is the product between this factor and the particle's radius. For example: with a particle radius of 2 and the inside radius factor of 0.8, the inside radius of the refinement zone would be 1.6 (see example below).
+* The ``refine mesh inside radius factor`` parameter defines the lower factor by which the effective radius will be multiplied to establish if a cell can be pre-refined. If the absolute distance to the surface is below :math:`(1 - \textit{factor}) * \textit{radius}`, then one of the two conditions is met. For example: with a particle radius of 2 and the inside radius factor of 0.8, the inside radius of the refinement zone would be 1.6 (see example below).
 
-* The ``refine mesh outside radius factor`` parameter defines the outside radius of the hypershell that forms the refinement zone around the particles. The radius used is the product between this factor and the particle's radius. For example: with a particle radius of 2 and the outside radius factor of 1.5, the outside radius of the refinement zone would be 3 (see example below).
+* The ``refine mesh outside radius factor`` parameter defines the higher limit of the near-particle refinement zone. If the absolute distance to the surface is below :math:`(\textit{factor} - 1) * \textit{radius}`, then the second condition is met. For example: with a particle radius of 2 and the outside radius factor of 1.5, the outside radius of the refinement zone would be 3 (see example below).
 
 .. warning::
-	When you want to use the hypershell refinement zone around particles, the mesh adaptation type used must be ``kelly``, otherwise no hypershell refinement will happen. See :doc:`../cfd/mesh_adaptation_control` for more details on adaptative mesh refinement.
+	When you want to use the near-particle refinement zone around particles, the mesh adaptation type used must be ``kelly``, otherwise no near-particle refinement will happen. See :doc:`../cfd/mesh_adaptation_control` for more details on adaptative mesh refinement.
```

Comment:
```suggestion
	The ``mesh adaptation type`` must be ``kelly`` to use the near-particle refinement zone around particles, otherwise, no near-particle refinement will happen. See :doc:`../cfd/mesh_adaptation_control` for more details on adaptative mesh refinement.
```